clang/test/SemaCXX/warn-unsafe-buffer-usage-libc-functions.cpp

```diff
@@ -4,7 +4,10 @@
 // RUN:            -verify %s -x objective-c++
 // RUN: %clang_cc1 -std=c++20 -Wno-all -Wunsafe-buffer-usage-in-libc-call \
 // RUN:            -verify %s
+// RUN: %clang_cc1 -std=c++20 -Wno-all -Wunsafe-buffer-usage \
+// RUN:            -verify -fexperimental-bounds-safety-attributes %s -DINTEROP_MODE
 
+#ifndef INTEROP_MODE
```

Comment:
Looks like the code isn't actually shared for INTEROP_MODE vs !INTEROP_MODE. Shouldn't it be just a separate test file?

Reply:
sure. These are things left un-upstreamed.